_posts/2023-11-28-how-to-create-custom-threat-detection-rules.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "Building custom threat detection rules with OpenSearch Security Analytics"
+authors:
+- xeniatup
+- jimishs
+- sandeshkr419
+date: 2023-11-28
+categories:
+ - feature
+ - technical-posts
+meta_keywords: threat detection, security analytics, SIEM, detection rules
```

Comment:
Please update the following: 

**meta_keywords:** OpenSearch security analytics, threat detection, security analytics plugin, detection rules